openlibrary/components/MergeUI/EditionSnippet.vue

```diff
@@ -112,6 +117,22 @@ export default {
                 this.edition.isbn_13 && ISBN.asIsbn10(this.edition.isbn_13),
             ].filter(x => x));
         }
+    },
+
+    methods: {
+        openEnlargedCover() {
+            let url = '';
+            if (this.cover_id) {
+                url = `https://covers.openlibrary.org/b/id/${this.cover_id}.jpg`;
+            } else {
+                const ocaid = this.edition.ocaid;
+                url = `https://archive.org/download/${ocaid}/page/cover_w600_h600.jpg`;
+            }
+
+            if (!url) return;
+
+            window.open(url, undefined, 'width=600,height=600');
```

Comment:
Why set the height and width? I'd assume if they're clicking to make it big they'd want it to be as big as possible?